index.html

```diff
@@ -1,317 +1,375 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WXBV13V4WN"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+      function gtag(){window.dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-WXBV13V4WN');
+    </script>
```

Comment:
- [ ] OPTIONAL
Let's keep our _html_ code clean by keeping **all** JavaScript code in a separate file